bin/qds.py

```diff
@@ -587,7 +587,7 @@ def main():
                          help="skip verification of server SSL certificate. Insecure: use with caution.")
 
     optparser.add_option("--cloud_name", dest="cloud_name",
-                         default=os.getenv('CLOUD_PROVIDER'),
+                         default=os.getenv('CLOUD_PROVIDER') or "AWS",
```

Comment:
better use getenv defaults
```suggestion
                         default=os.getenv('CLOUD_PROVIDER', "AWS"),
```

Reply:
If env level default is not present, then only it will move to hard coded value. Else the code simply fails as no cloud is defined.

Reply:
my bad, i read the comment wrong. Will change it.